src/main/java/seedu/address/model/todo/Todo.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.todo;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a task to be done.
+ * Guarantees: immutable
+ */
+public class Todo {
+    private final String description;
+
+    /**
+     * Constructs a Todo.
+     *
+     * @param description The description of this todo.
+     */
+    public Todo(String description) {
+        requireNonNull(description);
+        this.description = description;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    @Override
+    public String toString() {
+        return description;
+    }
+
+    /**
+     * Returns true if both Todos have the same description.
+     */
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        if (!(other instanceof Todo)) {
+            return false;
+        }
+
+        Todo otherTodo = (Todo) other;
```

Comment:
Might be better to use `else if`s to show that only one of the conditions will be run
Edit: Should we standardize to use short circuit implementation as per other classes?

Reply:
Oh I actually modified this from the Person.java class so that it's standardised across the models

If we wanna standardise to short circuit then I think have to edit the Person class as well?

Reply:
Okay, so it means that AB3 didn't standardise.. haha I'll change that in my PR then.

Reply:
I think we can just make a separate PR to address this issue!